src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java

```diff
@@ -1842,13 +1842,13 @@ public UploadSSCBlock(String appName, String appVersion) {
 
 		@Deprecated
 		public UploadSSCBlock(String projectName, String projectVersion, String filterSet, String searchCondition,
-							  String timeout, String pollingInterval) {
+							  Integer timeout, Integer pollingInterval) {
 			this.projectName = projectName != null ? projectName.trim() : "";
 			this.projectVersion = projectName != null ? projectVersion.trim() : "";
 			this.filterSet = filterSet != null ? filterSet.trim() : "";
 			this.searchCondition = searchCondition != null ? searchCondition.trim() : "";
-			this.timeout = timeout != null ? timeout.trim() : "";
-			this.pollingInterval = pollingInterval != null ? pollingInterval.trim() : "";
+			this.timeout = timeout;
```

Comment:
Question:
Doesn't need to validate negative number?

Reply:
We have the range validation for this field - https://github.com/jenkinsci/fortify-plugin/blob/master/src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java#L1077-L1092.